tests/webview/integration/test_rex_redirects.py

```diff
@@ -91,13 +103,47 @@ def test_chemistry_2e_uri_redirect_to_rex(webview_base_url, rex_base_url, chemis
     response = get_url(cnx_url)
 
     # THEN we are redirected to rex
-    if response.status_code != 200:
-        log_failures_to_csv(response.status_code, cnx_url)
+    subchapters = re.findall(r"(^\d+-\d+-\w+)", cnx_page_slug)
+    appendix = re.findall(r"(^appendix)", cnx_page_slug)
+    intro = re.findall(r"(^\d+-introduction)", cnx_page_slug)
+    key = re.findall(r"(.key*)", cnx_page_slug)
+    exercises = re.findall(r"(.exercises*)", cnx_page_slug)
+    summary = re.findall(r"(.summary*)", cnx_page_slug)
+    questions = re.findall(r"(.questions*)", cnx_page_slug)
+
+    if intro and response.status_code != 200:
```

Comment:
It seems like this `if, elif, elif, ...` could be simplified to:

```py
    is_interesting = intro or subchapters or appendix or key or exercises or summary or questions
    if is_interesting and response.status_code != 200:
        log_failures_to_csv(response.status_code, cnx_page_slug, cnx_url)
        pytest.fail(f"{response.status_code} in {cnx_page_slug}")
```

Reply:
Thanks @philschatz Done the changes